core/test/gather/mock-commands.js

```diff
@@ -30,16 +30,8 @@ import {fnAny} from '../test-utils.js';
  *      returns the protocol message argument.
  *
  * To mock an error response, use `send.mockResponse('Command', () => Promise.reject(error))`.
- *
- * There are two variants of sendCommand, one that expects a sessionId as the second positional
- * argument (legacy Lighthouse `Connection.sendCommand`) and one that does not (Fraggle Rock
- * `ProtocolSession.sendCommand`).
- *
- * @param {{useSessionId: boolean}} [options]
  */
-function createMockSendCommandFn(options) {
```

Comment:
`useSessionId` always needs to be `false` now that we removed the legacy runner, so we can just get rid of the options entirely